e2e-network/k8s/wait-for-chaincode.sh

```diff
@@ -5,10 +5,10 @@ peer=$2
 channel=$3
 chaincode=$4
 version=$5
-config=/home/great/work/new/fablo/fablo-target/fabric-config/fabric-k8/org1.yaml
```

Comment:
I have mixed feelings about that - are we able to provide a path than can be used both locally and in CI?

Reply:
Wasn't able to find a path that works on both environment,  I think passing the get config path operation to  ``find`` leaves it accurate in most cases.

Reply:
I believe `(cd "$TEST_TMP" && "$FABLO_HOME/fablo.sh" down)` should work. And we should support it this way

Reply:
But to be honest, a better way would be to specify config path as a parameter. It would make the script more versatile